.github/workflows/conformance-externalworkloads.yaml

```diff
@@ -149,7 +149,7 @@ jobs:
           CILIUM_INSTALL_DEFAULTS="${{ steps.default_vars.outputs.cilium_install_defaults }} \
             --cluster-name=${{ env.clusterName }} \
             --datapath-mode=tunnel \
-            --helm-set kubeProxyReplacement=strict"
+            --helm-set kubeProxyReplacement=true"
```

Comment:
In some places this option is set with `--helm-set-string`, but here and in `conformance-multi-pool.yaml` with just `--helm-set`. I vaguely recall that the string type is recognized automatically? Just wanted to double check.

Reply:
I am under the same impression, considered that CI is green, I think it should be fine.

Will keep an eye on this point still.